sources/packages/backend/workflow/test/test-utils/constants/mock-constants.ts

```diff
@@ -22,3 +22,7 @@ export const JOB_MESSAGE_RESULT_SUFFIX = "messageResult";
  * Identifier of subprocesses in the workflow.
  */
 export const PARENT_SUBPROCESSES_VARIABLE = "parentSubprocesses";
+/**
+ * Suffix that indicates if a task or subprocess was invoked by the workflow.
+ */
+export const JOB_PASSTHROUGH_SUFFIX = "passthrough";
```

Comment:
What was the intention having passthrough suffixed(in the bpmn output value of variable)? to follow the pattern of aligning with it's purpose?

Reply:
Without the suffix, we would be actually updating the entire mocked object that is now the root of the mock.
For instance, we would be replacing the below:
```json
 "verify_application_exceptions_task": {
    "result": {
      "applicationExceptionStatus": "Approved"
    }
  },
```

by

```json
 "verify_application_exceptions_task": "verify_application_exceptions_task"
```